integration-cli/docker_api_swarm_test.go

```diff
@@ -65,11 +65,8 @@ func (s *DockerSwarmSuite) TestAPISwarmInit(c *check.C) {
 	assert.Equal(c, info.LocalNodeState, swarm.LocalNodeStateActive)
 
 	// Current state restoring after restarts
-	d1.Stop(c)
-	d2.Stop(c)
-
-	d1.StartNode(c)
-	d2.StartNode(c)
+	d1.RestartNode(c)
```

Comment:
This is not equivalent to the original.

Reply:
Ah, you mean, now d1 starts again before d2 is stopped? 🤔 let me change

Reply:
updated and rebased